include/Inventory/Inventory.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by musique88 on 8/22/20.
+//
+#pragma once
+
+#include <array>
+
+namespace CC
+{
+	class Spell;
+	static const int NUMBER_OF_ELEMENTS = 7;
```

Comment:
Never put a static variable in a header. In this case, what you could have done is to put:
`extern const int NUMBER_OF_ELEMENTS;` in your header 
`const int NUMBER_OF_ELEMENTS = 7;` and in your cpp file (here Inventory.cpp)
This way, your constant is created only once and is accessed remotely with the extern keyword.

Reply:
The variable was put in a macro